index.html

```diff
@@ -39,7 +39,196 @@ <h1 class="my-name">Bassem Shams</h1>
         </p>
         <button class="button">Start collaboration</button>
       </section>
-      <footer class="lower"></footer>
+      <section class="lower"></section>
+      <!--the second part of the project-->
+      <nav class="container-projects">
+        <h2 class="header-project">projects</h2>
+        <section class="project">
+          <nav>
+            <img
+              class="image-project"
+              src="./assests/images/Rectangle21.png"
+              alt="project image"
+            />
+          </nav>
+          <nav class="details-project">
+            <p class="name-project">project name goes here</p>
+            <ul class="lang-group">
+              <li class="lang-project"><span>HTML/CSS</span></li>
+              <li class="lang-project">Ruby on rails</li>
+              <li class="lang-project">Javascript</li>
+            </ul>
+            <button class="button-project">
+              See this project
+              <span
+                ><img src="./assests/images/Union.png" alt="arrow image"
+              /></span>
+            </button>
+          </nav>
+        </section>
+        <section class="project">
+          <nav>
+            <img
+              class="image-project"
+              src="./assests/images/Rectangle211.png"
+              alt="project image"
+            />
+          </nav>
+          <nav class="details-project">
+            <p class="name-project">project name goes here</p>
+            <ul class="lang-group">
+              <li class="lang-project"><span>HTML/CSS</span></li>
+              <li class="lang-project">Ruby on rails</li>
+              <li class="lang-project">Javascript</li>
+            </ul>
+            <button class="button-project">
+              See this project
+              <span>
+                <img src="./assests/images/Union.png" alt="arrow image"
+              /></span>
+            </button>
+          </nav>
+        </section>
+        <section class="project">
+          <nav>
+            <img
+              class="image-project"
+              src="./assests/images/Rectangle212.png"
+              alt="project image"
+            />
+          </nav>
+          <nav class="details-project">
+            <p class="name-project">project name goes here</p>
```

Comment:
- [ ] Kindly wrap the project title in a `<hx>` (h2, h3, ...,h6) tag as this will improve the SEO of your webpage.